WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
Ah, that might be a good solution for ruby, but not sure about other languages that want to interact with wcif.

Splitting it out into a seconds and hundredths field feels grosser to me than centiseconds. I'd be interested to see how others feel about that, though. I've been wrong before!

It's good to know that the attempt value needs a better writeup. I'll do that when I'm back at a computer.